proto/node/akash/market/v1beta5/bid.proto

```diff
@@ -9,60 +9,66 @@ import "akash/market/v1/bid.proto";
 
 option go_package = "pkg.akt.dev/go/node/market/v1beta5";
 
-// Bid stores BidID, state of bid and price
+// Bid stores BidID, state of bid and price.
 message Bid {
   option (gogoproto.equal)            = false;
   option (gogoproto.goproto_stringer) = false;
 
-  // BidState is an enum which refers to state of bid
+  // BidState is an enum which refers to state of bid.
   enum State {
     option (gogoproto.goproto_enum_prefix) = false;
 
-    // Prefix should start with 0 in enum. So declaring dummy state
+    // Prefix should start with 0 in enum. So declaring dummy state.
     invalid = 0 [
         (gogoproto.enumvalue_customname) = "BidStateInvalid"
     ];
-    // BidOpen denotes state for bid open
+    // BidOpen denotes state for bid open.
     open = 1 [
         (gogoproto.enumvalue_customname) = "BidOpen"
     ];
-    // BidMatched denotes state for bid open
+    // BidMatched denotes state for bid open.
     active = 2 [
         (gogoproto.enumvalue_customname) = "BidActive"
     ];
-    // BidLost denotes state for bid lost
+    // BidLost denotes state for bid lost.
     lost = 3 [
         (gogoproto.enumvalue_customname) = "BidLost"
     ];
-    // BidClosed denotes state for bid closed
+    // BidClosed denotes state for bid closed.
     closed = 4 [
         (gogoproto.enumvalue_customname) = "BidClosed"
     ];
   }
 
+  // BidID stores owner and all other seq numbers.
+  // A successful bid becomes a Lease(ID).
   akash.market.v1.BidID id = 1 [
     (gogoproto.customname) = "ID",
     (gogoproto.nullable)   = false,
     (gogoproto.jsontag)    = "id",
     (gogoproto.moretags)   = "yaml:\"id\""
   ];
 
+  // State represents the state of the Bid.
   State state = 2 [
     (gogoproto.jsontag)  = "state",
     (gogoproto.moretags) = "yaml:\"state\""
   ];
 
+  // Price holds the pricing stated on the Bid.
   cosmos.base.v1beta1.DecCoin price = 3 [
     (gogoproto.nullable) = false,
     (gogoproto.jsontag)  = "price",
     (gogoproto.moretags) = "yaml:\"price\""
   ];
 
+  // CreatedAt is the block height at which the Bid was created.
   int64 created_at = 4 [
-    (gogoproto.jsontag)  = "state",
```

Comment:
@troian found this bug and fixed it here. Is it okay?

Reply:
nice catch!